go/cmd/vtbackup/vtbackup.go

```diff
@@ -312,12 +322,14 @@ func takeBackup(ctx context.Context, topoServer *topo.Server, backupStorage back
 		if err := mysqlctl.Backup(ctx, backupParams); err != nil {
 			return fmt.Errorf("backup failed: %v", err)
 		}
+		durationByPhase.Set("InitialBackup", int64(time.Since(backupParams.BackupTime).Seconds()))
 		log.Info("Initial backup successful.")
 		return nil
 	}
 
 	backupDir := mysqlctl.GetBackupDir(initKeyspace, initShard)
 	log.Infof("Restoring latest backup from directory %v", backupDir)
+	restoreAt := time.Now()
```

Comment:
Just an idea : why not time.Now.Unix ?

Reply:
It never occurred to me! I have seen the `s := time.Now() ... time.Since(s).Seconds()` in use elsewhere in the `mysqlctl` package, so I just went with that:

https://github.com/vitessio/vitess/blob/0a173bed6f08eceba38e06269e004ce5c76ee96d/go/vt/mysqlctl/backup.go#L149

I'll go ahead and make the switch to `time.Now().Unix()`.

Reply:
Hm actually it looks like `time.Since` expects a `time.Time`, and `Unix()` returns an `int64` so it doesn't fit well into the pattern here.

Reply:
hmm, i think its ok .. time.unix make integer manipulation easy . in terms of functional sense it will not make difference.